aws/table_aws_docdb_cluster.go

```diff
@@ -0,0 +1,369 @@
+			},
+			{
+				Name:        "enabled_cloudwatch_logs_exports",
+				Description: "A list of log types that this cluster is configured to export to Amazon CloudWatch Logs.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "members",
+				Description: "A list of instances that make up the cluster.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("DBClusterMembers"),
+			},
+			{
+				Name:        "read_replica_identifiers",
+				Description: "A list of identifiers of the read replicas associated with this cluster.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "vpc_security_groups",
+				Description: "A list of VPC security groups that the DB cluster belongs to.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags attached to the Cluster.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getDocDBClusterTags,
+				Transform:   transform.FromField("TagList"),
+			},
+
+			// Standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getDocDBClusterTags,
+				Transform:   transform.FromField("TagList").Transform(docDBClusterTagListToTurbotTags),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DBClusterIdentifier"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("DBClusterArn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listDocDBClusters(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+
+	// Create Session
+	svc, err := DocDBService(ctx, d)
+	if err != nil {
+		logger.Error("aws_docdb_cluster.listDocDBClusters", "service_creation_error", err)
+		return nil, err
+	}
+
+	// Reduce the basic request limit down if the user has only requested a small number of rows
+	maxLimit := int32(100)
+	if d.QueryContext.Limit != nil {
+		limit := int32(*d.QueryContext.Limit)
+		if limit < maxLimit {
+			if limit < 20 {
+				maxLimit = 20
+			} else {
+				maxLimit = limit
+			}
+		}
+	}
+
+	input := docdb.DescribeDBClustersInput{
+		MaxRecords: aws.Int32(maxLimit),
+	}
+
+	paginator := docdb.NewDescribeDBClustersPaginator(svc, &input, func(o *docdb.DescribeDBClustersPaginatorOptions) {
+		o.Limit = maxLimit
+		o.StopOnDuplicateToken = true
+	})
+
+	for paginator.HasMorePages() {
+		output, err := paginator.NextPage(ctx)
+		if err != nil {
+			plugin.Logger(ctx).Error("aws_docdb_cluster.listDocDBClusters", "api_error", err)
+			return nil, err
+		}
+
+		for _, cluster := range output.DBClusters {
+
+			if helpers.StringSliceContains(
```

Comment:
```suggestion
			// The DescribeDBClusters API returns non-DocDB clusters as well,
			// but we only want DocDB clusters here.
			if helpers.StringSliceContains(
```

Only add this comment if non-DocDB clusters are returned too by default